### PR TITLE
Bug 1755 - Hook up remix buttons

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/87206f698916c08/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/7eb5cb896fde0a1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/87206f698916c08/output.txt

Total script time: 1.44 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/7eb5cb896fde0a1/output.txt

Total script time: 0.84 mins

Server is running on http://pr932.butyr.org/templates/basic/
Run the Test Suite: http://pr932.butyr.org/test/

File: cornfield/views/embed.jade [hunk 1]
```diff
@@ -50,8 +48,8 @@ html(lang='en')
             .embed-project
               div
                 .mozpopcorn mozilla Popcorn
```
Comment (mjschranz) [inline cornfield/views/embed.jade]: Unrelated to your fixes, but Shouldn't this be <pre>Mozilla Popcorn</pre>?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/2ba230e71b3945f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/21f3b946bb88d67/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2ba230e71b3945f/output.txt

Total script time: 1.26 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/21f3b946bb88d67/output.txt

Total script time: 0.65 mins

Server is running on http://pr932.butyr.org/templates/basic/
Run the Test Suite: http://pr932.butyr.org/test/

**Pull request merged**